Assets/Scripts/PlayerController/PlayerMovement.cs

```diff
@@ -37,6 +37,7 @@ public class PlayerMovement : MonoBehaviour
     private float m_LandingTime = 0;
     private bool m_IsMidAir;
     private bool m_JumpedInCurrentFrame;
+    private bool m_cannotMove;
```

Comment:
Also not a big deal, but the usual convention here is `m_StartsWithCaps`, so maybe `m_CannotMove` rather than `m_cannotMove`